### PR TITLE
Feature/sc 15150/add pre commit to terraform module

### PR DESCRIPTION
**Pull request by pmanal1**
Completes [15150](https://app.shortcut.com/clarkcan/story/15150/add-pre-commit-to-terraform-module). This story creates a new pre-commit strategy using terraform hooks made for the pre-commit framework.

Notes:
- I've removed tflint from the local and CI environments. Should we keep tflint? This new pre-commit strategy has a tflint hook that shouldn't be too hard to add.
- Changed Makefile and Readme comments to adapt to new strategy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,3 @@
-# Terraform Module
+<!-- BEGIN_TF_DOCS -->
```
Comment (pzalep1) [inline README.md:1]: Why was this deleted?

**Review by pzalep1 — COMMENTED** [review]
Just the one question

**Review by JosueMolinaMorales — APPROVED** [review]

File: script.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+set -e
+myArray=("tflint" "terraform-fmt")
+
```
Comment (Cwagne17) [inline script.sh:3]: we can remove this file

**Review by Cwagne17 — COMMENTED** [review]
Looks good, lets remove the two files and pull main.

**Review by pzalep1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Review by Cwagne17 — APPROVED** [review]
